### PR TITLE
[doc] Add links to `pipectl init` from the pages of configuring ECS/K8s App

### PR DESCRIPTION
**Pull request by t-kikuc**
**What this PR does / why we need it**:

To Inform users `pipectl init`.

**Which issue(s) this PR fixes**:

**Does this PR introduce a user-facing change?**: no

- **How are users affected by this change**: no
- **Is this breaking change**: no
- **How to migrate (if breaking change)**: no

### CODE REVIEW

File: docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ To deploy your application to run continuously or to place it behind a load bala
 
 If you're not familiar with ECS, you can get examples for those files from [here](../../../../examples/#ecs-applications).
 
+## Easier configuration by `pipectl init`
+
+You can generate an application config file easily and interactively by [`pipectl init`](../../command-line-tool.md#generating-an-application-config-apppipecdyaml).
+
+
```
Comment (khanhtc1202) [inline docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md:26]:
```suggestion
Note:

You can generate an application config file quickly and interactively by [`pipectl init`](../../command-line-tool.md#generating-an-application-config-apppipecdyaml).


```

This should be a note, not a new session. wdyt?

File: docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/kubernetes.md [hunk 2]
```diff
@@ -8,6 +8,11 @@ description: >
 
 Based on the application configuration and the pull request changes, PipeCD plans how to execute the deployment: doing quick sync or doing progressive sync with the specified pipeline.
 
+## Easier configuration by `pipectl init`
+
+You can generate an application config file easily and interactively by [`pipectl init`](../../command-line-tool.md#generating-an-application-config-apppipecdyaml).
+
+
```
Comment (khanhtc1202) [inline docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/kubernetes.md:15]: Same as above

File: docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md (hunk 1 above)
Comment (t-kikuc) [reply docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md]: @khanhtc1202 
Thanks, I fixed all

File: docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/kubernetes.md (hunk 2 above)
Comment (t-kikuc) [reply docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/kubernetes.md]: fixed

**Review by khanhtc1202 — APPROVED** [review]
LGTM 👍

**Pull request merged**